### ISSUE TITLE
Accessibility improvement: In chat screen, "Copy to Clipboard" mouseover button is inaccessible using tap and arrow keys

### ISSUE THREAD

**Issue by arielgreen** [Expensify/Expensify.cash#3103]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Expected Result:
In chat screen, "Copy to Clipboard" mouseover button should be accessible using tap and arrow keys.
## Actual Result:
"Copy to Clipboard" mouseover button is inaccessible using tap and arrow keys.
## Action Performed:
1. Launch the app/webapp
2. Prerequisite: Turn ON the Screen reader.
Web:Navigate the page using tap and arrow keys then verify.
Mobile:Navigate using Swipe gesture(From Left to Right) and verify.

## Platform:
Windows/JAWS/Chrome
MACOs/Voiceover/Safari
Android/Talkback
iOS/Voiceover
MacWebapp/Voiceover

**Notes/Photos/Videos:** 
https://user-images.githubusercontent.com/37308300/119419308-ffbca900-bcc7-11eb-9432-4ccbe187ac22.mp4
![image](https://user-images.githubusercontent.com/37308300/119419376-1c58e100-bcc8-11eb-8b09-6ed9ad707b3b.png)

[View all open jobs on Upwork](https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2)

**Comment by MelvinBot** [comment]
Triggered auto assignment to @kadiealexander (`External`), see https://stackoverflow.com/c/expensify/questions/8582 for more details.

**Comment by MelvinBot** [comment]
Triggered auto assignment to @alexpensify (`AutoAssignerTriage`), see https://stackoverflow.com/c/expensify/questions/4749 for more details.

**Comment by kadiealexander** [comment]
Popping this through the triage process!

**Comment by MelvinBot** [comment]
Triggered auto assignment to @jasperhuangg (`Engineering`), see https://stackoverflow.com/c/expensify/questions/4319 for more details.

**Comment by jasperhuangg** [comment]
This seems to work fine on mobile and should just be an issue on Web/Desktop.

**Comment by marcaaron** [comment]
Hey @arielgreen, I'm curious is this part of a larger effort to make E.cash more accessible and can I learn more about that somewhere?

**Comment by arielgreen** [comment]
Hey @marcaaron, yep; @laurenreidexpensify is heading this up. See https://github.com/Expensify/Expensify/issues/164705, #accessibility in Slack, and specifically [this thread](https://expensify.slack.com/archives/CNKBZSQ07/p1621529612003300), which precipitated these issues.
